### ISSUE TITLE
Cannot create prop mappings with dashes

### ISSUE THREAD

**Issue by leoweigand** [figma/code-connect#97]
We have an input component that accepts some props matching the corresponding HTML attributes, in this example `aria-invalid`. We can publish the component but see "Failed to load Code Connect example" on Figma. Changing the name to `ariaInvalid` makes the error go away, indicating that it is in fact the casing that causes this issue.

Code connect version: `1.0.2`

Code example:

```ts
figma.connect(
  Input,
  'https://www.figma.com/...',
  {
    props: {
      label: figma.string('Aa → Label'),
      'aria-invalid': figma.enum('State', {
        Default: undefined,
        Filled: undefined,
        Active: undefined,
        Error: true,
      }),
    },
    example: (props) => <Input {...props} />,
  },
)
```

**Comment by ptomas-figma** [comment]
This has now been fixed in the new version of Code Connect.
